clients/gitlabrepo/repo.go

```diff
@@ -46,6 +47,9 @@ type repoURL struct {
 The following input format is not supported:
 	* https://gitlab.<companyDomain:string>.com/projects/<projectID:int>
 */
+
+var errInvalidGitlabRepoURL = errors.New("repo is not a gitlab repo")
+
```

Comment:
can we move this higher so we don't get between the godoc and function for `parse`?